elastalert/elastalert.py

```diff
@@ -859,6 +859,14 @@ def enhance_filter(self, rule):
             filters.append({'query': query_str_filter})
         elastalert_logger.debug("Enhanced filter with {} terms: {}".format(listname, str(query_str_filter)))
 
+    def get_elasticsearch_client(self, rule):
+        key = rule['name']
+        es_client = self.es_clients.get(key)
+        if es_client is None:
+            es_client = elasticsearch_client(rule)
+            self.es_clients[key] = es_client
```

Comment:
Could be condensed into a single line,

```
self.es_clients[key] = es_client = elasticsearch_client(rule)
```

Not sure what your stylistic preference is.

Reply:
I prefer readability over condensed code, so what you have looks fine.